builtin/providers/aws/resource_aws_elasticache_cluster.go

```diff
@@ -261,6 +289,12 @@ func resourceAwsElasticacheClusterRead(d *schema.ResourceData, meta interface{})
 		d.Set("security_group_ids", c.SecurityGroups)
 		d.Set("parameter_group_name", c.CacheParameterGroup)
 		d.Set("maintenance_window", c.PreferredMaintenanceWindow)
+		if c.SnapshotWindow != nil {
```

Comment:
`d.Set` can handle `nil`s, so this guard isn't necessary. It would only be necessary if you're wanting to log our their value